script/Ciel/Day 14/14_01B_03_CIEL14B_5.txt

```diff
@@ -106,7 +106,7 @@ But... will I even be able to?
 [sha:47454a1684a1a2dc16f578e182bd28d4db349c90]{
 -- Page 177, Offset 40768.
 -- あのデカブツを追い掛け、追い越して、シエル先輩が弾き飛ばされた校舎に辿り着く事が。
-I'd have to chase after that giant, overtake it,
+I'd have to chase after that Goliath, overtake it,
```

Comment:
I honestly think goliath shouldn't be capitalised. Not talking about 'the' Goliath, just a massive being.